CHANGELOG.md

```diff
@@ -1,3 +1,221 @@
+- Cleanup Terraform providers before Terraform 0.13 upgrades
+  ([#860](https://github.com/kinvolk/lokomotive/pull/860)).
+- kubelet e2e: Enable the disruptive test
+  ([#1012](https://github.com/kinvolk/lokomotive/pull/1012)).
+- .golangci.yml: Re-enable linters
+  ([#1029](https://github.com/kinvolk/lokomotive/pull/1029)).
+- Fix scripts/find-updates.sh
+  ([#1034](https://github.com/kinvolk/lokomotive/pull/1034)),
+  ([#1068](https://github.com/kinvolk/lokomotive/pull/1068)),
+  ([#1080](https://github.com/kinvolk/lokomotive/pull/1080)).
+- pkg/terraform: improvements
+  ([#1027](https://github.com/kinvolk/lokomotive/pull/1027)).
+- cli/cmd: cleanups part 1
+  ([#1013](https://github.com/kinvolk/lokomotive/pull/1013)).
+- test/components/kubernetes: remove kubelet pod when testing node labels
+  ([#1052](https://github.com/kinvolk/lokomotive/pull/1052)).
+- remove usage of template_file
+  ([#1046](https://github.com/kinvolk/lokomotive/pull/1046)).
+- test/*: de-duplicate value timeout and retryInterval
+  ([#1049](https://github.com/kinvolk/lokomotive/pull/1049)).
+- packet: Read BGP peer address from metadata service
+  ([#1010](https://github.com/kinvolk/lokomotive/pull/1010)).
+- pkg/assets: cleanup exported API
+  ([#936](https://github.com/kinvolk/lokomotive/pull/936)).
+- cobra upgrade to v1.1.1
+  ([#1082](https://github.com/kinvolk/lokomotive/pull/1082)),
+  ([#1091](https://github.com/kinvolk/lokomotive/pull/1091)).
+- cli/cmd: cleanups part 2
+  ([#1015](https://github.com/kinvolk/lokomotive/pull/1015)).
+- Add github actions
+  ([#1074](https://github.com/kinvolk/lokomotive/pull/1074)).
+- Makefile: use latest Go when building in Docker
+  ([#1083](https://github.com/kinvolk/lokomotive/pull/1083)).
+- cli/cmd: cleanups part 3
+  ([#1018](https://github.com/kinvolk/lokomotive/pull/1018)).
+- Add new CI config for Packet based FLUO testing
+  ([#1110](https://github.com/kinvolk/lokomotive/pull/1110)).
+
+### Upgrading from v0.4.1
+
+#### Configuration syntax changes
+
+There have been some minor changes to the configurations of worker nodes, `labels` and `taints` format changed from `string` to `map(string)` for AWS and Packet platform.
+
+old:
+
+```
+labels = "testing=true"
+
+taints = "nodeType=storage:NoSchedule"
+```
+
+new:
+
+```
+labels = {
+  "testing" = "true"
+}
+
+taints = {
+  "nodeType" = "storage:NoSchedule"
+}
+```
+
+#### Cluster upgrade steps
+
+Due to [Terraform](https://github.com/kinvolk/lokomotive/pull/824) and [Kubernetes](https://github.com/kinvolk/lokomotive/pull/1030) upgrades to v0.13+ and v1.19.3 respectively, there are some additional steps that you need to follow. Go to your cluster's directory and follow these steps:
+
+```
+1. Upgrade to Terraform 0.13 binary locally. Follow https://www.terraform.io/upgrade-guides/0-13.html.
+
+2. export PLATFORM="packet" && export ASSETS_DIR="assets" # export your platform name and assets directory name used in your platform configuration.
```

Comment:
Why this is all in "````"? I suggest we use regular list and move shell commands into "```sh" blocks as content of the list item.

```suggestion
2. Export your platform name and assets directory name used in your platform configuration. It will be used in next steps.
  ```sh
  export PLATFORM="packet" && export ASSETS_DIR="assets"
  ```
```